ckanext/unaids/react/components/FileInputComponent/src/FileUploader.js

```diff
@@ -34,10 +37,28 @@ export default function FileUploader({
     const { getRootProps, getInputProps, open } = useDropzone({
         multiple: false,
         noClick: true,
+        maxSize: maxResourceSize * 1000000,
         onDrop: acceptedFiles =>
             handleFileSelected(acceptedFiles[0]),
-        onDropRejected: rejectedFiles =>
-            handleFileSelected(rejectedFiles[0].file),
+        onDropRejected: rejectedFiles => {
+            if (rejectedFiles.length > 1) {
+                setUploadError({
+                    error: ckan.i18n._('Too many files'),
+                    description: ckan.i18n._('You can only upload one file for each resource.')
+                });
```

Comment:
Is this even possible scenario once we have set `multiple: false`?

Reply:
Yep, as it's still possible to drag/drop multiple files into the dropzone